Gemfile.docker

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+# source 'https://gem.fury.io/enderahmetyurt/' do
+#   gem 'rspotify'
+# end 
```

Comment:
we can remove this instead commenting it

Reply:
removed Gemfile.docker file